chainguard-dev/edu
# Adding tags to open source directory files and software security dire…

…ctory files

## Type of change
Added tags to content in open source directory and software security directory. 

### What should this PR do?


### Why are we making this change?
Tags will help with user discovery and internal metrics. 

### What are the acceptance criteria? 
Tags should appear on relevant content and nothing should break. 

### How should this PR be tested?
Check out a few of the tutorials in the open source directory and the software security directory and make sure tags appear.
